### PR TITLE
Log error on open document widget.

### PR DESCRIPTION
**Pull request by trungleduc**
## References

On document opening, the widget is disposed if the context initialization fails.  Since no error is logged, debugging the issue from the external extension is very hard.

## Code changes
Log the error in the try-catch block.

## User-facing changes
N/A

## Backwards-incompatible changes

N/A

### CODE REVIEW

File: packages/docmanager/src/manager.ts [hunk 1]
```diff
@@ -592,6 +592,7 @@ export class DocumentManager implements IDocumentManager {
 
     // If the initial opening of the context fails, dispose of the widget.
     ready.catch(err => {
+      console.error(err)
```
Comment (fcollonval) [inline packages/docmanager/src/manager.ts:595]: Adding some context to the error would be nice

```suggestion
      console.error(`Failed to initialize the context with '${factory.name}' for ${path}`, err)
```

**Review by fcollonval — CHANGES_REQUESTED** [review]

**Comment by fcollonval** [comment]
Could you fix the linter @trungleduc ?

**Comment by trungleduc** [comment]
Done

**Comment by jtpio** [comment]
@meeseeksdev please backport to 3.3.x

**Pull request merged**